.circleci/config.yml

```diff
@@ -229,6 +229,10 @@ jobs:
           name: Run e2e profileFeatures test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="profileFeatures" --prod_env
+      - run:
```

Comment:
Just curious -- have we tried running these on github actions?

Reply:
No